src/olympia/reviewers/templatetags/code_manager_tags.py

```diff
@@ -0,0 +1,19 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag
+def code_manager_url(
+        page, addon_id, version_id, compare_version_id=None, file=None):
```

Comment:
Is `compare_version_id` the `base_version_id`? We use that in code-manager so would be nice to keep the same name.